samples/excel/89-preview-apis/pivot-table-filters-and-summaries.yaml

```diff
@@ -0,0 +1,213 @@
+                await context.sync();
+
+                pivotTable.dataHierarchies.items[0].summarizeBy = functionType;
+                pivotTable.dataHierarchies.items[1].summarizeBy = functionType;
+                await context.sync();
+            });
+        }
+
+        async function getCrateTotal() {
+            await Excel.run(async (context) => {
+                let pivotTable = context.workbook.worksheets.getActiveWorksheet().pivotTables.getItem("Farm Sales");
+                let range = pivotTable.layout.getDataBodyRange();
+                let grandTotalRange = range.getLastRow();
+                grandTotalRange.load();
+                await context.sync();
+                
+                let masterTotalRange = context.workbook.worksheets.getActiveWorksheet().getRange("B27:C27");
+                masterTotalRange.formulas = [["All Crates", "=SUM(" + grandTotalRange.address + ")"]];
+                await context.sync();
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                const dataSheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Data");
+                const pivotSheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Pivot");
+
+                const data = [["Farm", "Type", "Classification", "Crates Sold at Farm", "Crates Sold Wholesale"],
+                ["A Farms", "Lime", "Organic", 300, 2000],
+                ["A Farms", "Lemon", "Organic", 250, 1800],
+                ["A Farms", "Orange", "Organic", 200, 2200],
+                ["B Farms", "Lime", "Conventional", 80, 1000],
+                ["B Farms", "Lemon", "Conventional", 75, 1230],
+                ["B Farms", "Orange", "Conventional", 25, 800],
+                ["B Farms", "Orange", "Organic", 20, 500],
+                ["B Farms", "Lemon", "Organic", 10, 770],
+                ["B Farms", "Kiwi", "Conventional", 30, 300],
+                ["B Farms", "Lime", "Organic", 50, 400],
+                ["C Farms", "Apple", "Organic", 275, 220],
+                ["C Farms", "Kiwi", "Organic", 200, 120],
+                ["D Farms", "Apple", "Conventional", 100, 3000],
+                ["D Farms", "Apple", "Organic", 80, 2800],
+                ["E Farms", "Lime", "Conventional", 160, 2700],
+                ["E Farms", "Orange", "Conventional", 180, 2000],
+                ["E Farms", "Apple", "Conventional", 245, 2200],
+                ["E Farms", "Kiwi", "Conventional", 200, 1500],
+                ["F Farms", "Kiwi", "Organic", 100, 150],
+                ["F Farms", "Lemon", "Conventional", 150, 270]];
+
+                const range = dataSheet.getRange("A1:E21");
+                range.values = data;
+                range.format.autofitColumns();
+                pivotSheet.activate();
+
+                context.workbook.worksheets.getActiveWorksheet()
```

Comment:
Does the sheet name "Pivot" need to be in the third parameter of `pivotTables.add()`?  You've activated the Pivot sheet in line 108 and you are calling the method for the pivotTables of the current active sheet.